pytest/requirements.txt

```diff
@@ -2,7 +2,8 @@ PyGithub
 base58
 cython
 deepdiff
-ed25519
+# TODO: replace ed25519 package with https://github.com/pyca/pynacl
```

Comment:
can we do this now, given that https://github.com/near/nearcore/pull/11624/files moves the implementation to use pyca?